docs/Technical-Documentation/Architecture-Decision-Record/017-elastisearch.md

```diff
@@ -0,0 +1,29 @@
+# 17. Elastisearch Architecture
+
+Date: 2021-01-12 (yyyy-mm-dd)
+
+## Status
+
+Accepted
+
+## Context
+
+To modernize the eventual reporting stack for TDP, we have a need for a usable data backend.
+
+## Decision
+
+We will use an Elastisearch and Kibana stack for their modern feature set and scalability with large data sets. The Elastisearch/Kibana cluster will be an application hosted in Cloud.gov with the rest of our TDP application so we can leverage all the work surrounding authentication, security, and data compliance.
+
+## Consequences
+
+### Pros
+ - Elastisearch is fast becoming the industry standard if not already the standard for large data sets and getting their feature sets for customizable queries, scalability, and advanced storage methods.
```

Comment:
also worth mentioning the security- and integrity- related mitigation this solution offers, since there have been concerns about how OFA data staff would access the db and still authenticate via PIV/CAC